IncidentFields/incidentfield-asn.json

```diff
@@ -0,0 +1,46 @@
+{
+  "sortValues": null,
+  "closeForm": false,
+  "cliName": "asn",
+  "neverSetAsRequired": false,
+  "threshold": 72,
+  "defaultRows": null,
+  "id": "indicator_asn",
+  "validatedError": "",
+  "group": 2,
+  "script": "",
+  "isReadOnly": false,
+  "commitMessage": "",
+  "system": false,
+  "content": false,
+  "unsearchable": false,
+  "version": -1,
+  "unmapped": false,
+  "hidden": false,
+  "type": "shortText",
+  "breachScript": "",
+  "editForm": true,
+  "description": "",
+  "associatedToAll": false,
+  "shouldCommit": false,
+  "shouldPublish": false,
+  "associatedTypes": [
+    "IP",
+    "IP_tmp"
```

Comment:
@DeanArbel what is this type?

Reply:
that's a placeholder from before export was added, I'll remove it